defaults/test_default.yaml

```diff
@@ -227,3 +227,13 @@ run_fullscan: []
 
 stress_step_duration: '15m'
 simulated_racks: 0
+
+scylla_network_config:
+- address: "listen_address"
+  ip_type: “ipv4”
```

Comment:
let drop the ", seems like we have different types of it here.
and it's not mandatory in yamls

Reply:
removed